p/vbn/products.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+module.exports = [
+	{
+		id: 'express-train',
+		mode: 'train',
+		bitmasks: [1],
+		name: 'InterCityExpress',
+		short: 'ICE',
+		default: true
+	},
+	{
+		id: 'national-train',
+		mode: 'train',
+		bitmasks: [2],
+		name: 'InterCity, EuroCity, CityNightLine',
+		short: 'IC/EC/CNL',
+		default: true
+	},
+	{
+		id: 'national-train-2',
```

Comment:
I think this id is sub-optimal. We should either group national trains or give this one a different id.